### PR TITLE
Adding TrainerRoad app

### PR DESCRIPTION
**Pull request by KevinLaMS**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94239)

### CODE REVIEW

**Comment by KevinLaMS** [comment]
The URL is accurate.  
It comes from the main website:  https://www.trainerroad.com/download/
You will notice that the domain is similar as well in that it starts with Trainrd:
https://**train**rdtrcmn01un1softw01.blob.core.windows.net/

File: manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TrainerRoad.TrainerRoad
+PackageVersion: 2023.1.2.166
+PackageLocale: en-US
+Publisher: TrainerRoad LLC
+PackageName: TrainerRoad
```
Comment (stephengillie) [inline manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.trainerroad.com/
PublisherSupportUrl: https://support.trainerroad.com/hc/en-us
PrivacyUrl:   https://www.trainerroad.com/privacy-policy/
PackageName: TrainerRoad
PackageUrl: https://www.trainerroad.com/download/
```

File: manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TrainerRoad.TrainerRoad
+PackageVersion: 2023.1.2.166
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://trainrdtrcmn01un1softw01.blob.core.windows.net/installers/windows/v001/Production/TrainerRoad.exe
+  InstallerSha256: 752F97E089D3CF31A85EE4F1E1F7F9B51E676FA18A72405DB7218B5B20709442
```
Comment (stephengillie) [inline manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.installer.yaml:10]:
```suggestion
  InstallerSha256: A01164CD6BB9C936E5CDBA9FF762612D6B267977B5C431D21979A20F19E74B58
```

Comment (stephengillie) [reply manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.installer.yaml:10]: Hi @KevinLaMS,

Apparently, the developer has modified the package. Possibly with a bugfix.

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\t\TrainerRoad\TrainerRoad\2023.1.2.166
>   
>   * https://support.trainerroad.com/hc/en-us
>     
>     * Http status code: Forbidden

This is likely a datacenter block. The page works fine from my location.

**Comment by stephengillie** [comment]
Closing due to: No Recent Activity

**Pull request closed without merging**